index.js

```diff
@@ -155,23 +147,35 @@ function createQuestions(config) {
  * Format the git commit message from given answers.
  *
  * @param {Object} answers Answers provide by `inquier.js`
+ * @param {Object} config Result of the `getConfig` returned promise
  * @return {String} Formated git commit message
  */
-function format(answers) {
+
+function format(answers, config) {
+  const format = config.format
```

Comment:
cf. comment about `conventional`. When set to `true` it should take precedence over the `config.format`:
```suggestion
  const format = config.conventional
    ? `{type}{scope}: {emoji} {subject}`
    : config.format
```